tests/unit/plugins/modules/test_vm_nic.py

```diff
@@ -200,7 +200,6 @@ def test_minimal_set_of_params(self, run_main_with_reboot, mocker):
         assert results == {
             "changed": False,
             "records": {},
```

Comment:
Should there be an empty list? In the case of ensure_absent() an empty list is returned

Reply:
I guess we have incorrectly mocked return values. I do not want to extend this PR with fixing unittests.